devnet/eth-devnet/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:22.04
+
+# Install dependencies
+RUN apt-get update && \
+    apt-get install -y software-properties-common wget curl libgomp1 build-essential git gnupg openssl && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Node.js (required for Bundler)
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - && \
+    apt-get install -y nodejs && \
+    npm install -g yarn
+
+# Set working directory
+WORKDIR /app
+
+# Copy binaries and scripts into the image
+COPY geth \
```

Comment:
please see https://github.com/eth-infinitism/go-ethereum/blob/RIP-7560-revision-2/Dockerfile
on how to creater a "builder" docker image, so that the final image is lighter 
(also, they compile geth as static image, and thus don't have to copy shared objects, only a single binary)